web/server/codechecker_server/api/mass_store_run.py

```diff
@@ -1189,7 +1191,7 @@ def store(self) -> int:
                                       self.__name, run_lock.locked_at)
 
                             # Actual store operation begins here.
-                            run_id = self.__add_checker_run(
+                            run_id, update_run = self.__add_checker_run(
```

Comment:
Maybe `__add_or_update_run()` is a better name instead of `__add_checker_run()`.